PodrezHw18/PodrezHw18/ViewController.swift

```diff
@@ -0,0 +1,125 @@
+//
+//  ViewController.swift
+//  PodrezHw18
+//
+//  Created by Oleg Podrez on 26.08.24.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+    
+    
+    let label: UILabel = {
+        let label = UILabel()
+        label.text = ""
+        label.numberOfLines = 0
+        label.textColor = .black
+        label.textAlignment = .center
+        label.font = UIFont.systemFont(ofSize: 30)
+        return label
+    }()
+    
+    let textField: UITextField = {
+        let textField = UITextField()
+        textField.placeholder = "Передать тут"
+        textField.borderStyle = .roundedRect
+        textField.backgroundColor? = .clear
+        return textField
+    }()
+    
+    let button: UIButton = {
+        let button = UIButton(type: .system)
+        button.setTitle("Нажать тут", for: .normal)
+        button.addTarget(self, action: #selector(buttonTapped), for: .touchUpInside)
+        return button
+    }()
+    
+    let notificationView: UIView = {
+        let notificationView = UIView()
+        notificationView.backgroundColor = .white
+        notificationView.layer.cornerRadius = 15
+        notificationView.layer.shadowColor = UIColor.systemOrange.cgColor
+        notificationView.layer.shadowOpacity = 0.3
+        notificationView.layer.shadowOffset = CGSize(width: 6, height: 2)
+        notificationView.layer.shadowRadius = 4
+        return notificationView
+    }()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .systemCyan
+        
+        setupView()
+        
+        let tapGesture = UITapGestureRecognizer(target: self, action: #selector(dismissKeyboard))
+        view.addGestureRecognizer(tapGesture)
+        
+        
+        NotificationCenter.default.addObserver(self, selector: #selector(keyboardDidShow), name: UIResponder.keyboardDidShowNotification, object: nil)
```

Comment:
Старайся не писать такие длинные строки, при таких длинных вызовах можно каждый новый аргумент на новой строке писать
```suggestion
        NotificationCenter.default.addObserver(
            self, 
            selector: #selector(keyboardDidShow), 
            name: UIResponder.keyboardDidShowNotification, 
            object: nil
        )
```